app/src/main/java/com/example/dynamicdiet/ui/theme/Theme.kt

```diff
@@ -28,7 +28,10 @@ private val LightColorPalette = lightColors(
 )
 
 @Composable
-fun DynamicDietTheme(darkTheme: Boolean = isSystemInDarkTheme(), content: @Composable () -> Unit) {
+fun DynamicDietTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
```

Comment:
just make code break line, look a bit of clean.

Reply:
The IDE can handle formatting automatically, with Control Alt Shift L.  I'd leave it to the IDE on this one.

Reply:
thanks. That's good to know.